components/modal.html

```diff
@@ -0,0 +1,209 @@
+                style="
+                  width: 100%;
+                  height: 200px;
+                  border: none;
+                  transform: scale(1);
+                  overflow: hidden;
+                "
+                sandbox="allow-scripts allow-same-origin"
+              >
+              </iframe>
+            </code>
+          </section>
+          <footer class="nav-footer flex jc-between">
+            <a href="./navigation.html">
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M8.5 19l-7-7  7-7 "
+                  stroke="var(--secondary-color)"
+                  stroke-width="1.5"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                ></path>
+              </svg>
+              Navigation
+            </a>
+            <a href="./rating.html"
+              >Rating
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M8.5 5l7 7-7 7"
+                  stroke="var(--secondary-color)"
+                  stroke-width="1.5"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                ></path>
+              </svg>
+            </a>
+          </footer>
+        </aside>
+        <!-- Documentation of component end -->
+      </section>
+      <!-- main document ends here -->
+    </main>
+    <script type="text/javascript">
+      console.log("hello world");
+      const modalBtn = document.getElementById("modal-btn");
+      const modal = document.getElementById("modal");
+      const closeModal = document.getElementById("close-modal");
+      const closeModal_2 = document.getElementById("close-modal_2");
+      modalBtn.addEventListener("click", () => {
+        modal.classList.add("active");
+      });
+      closeModal_2.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
+      closeModal.addEventListener("click", () => {
+        modal.classList.remove("active");
+      });
+    </script>
```

Comment:
Suggestion : the JS Code in a separate file for code reusability and separation of the logic from design

Reply:
I am just using it here coz it is a small piece so that why I am keeping it here.